packages/framework/src/Component/GrapesJs/GrapesJsParser.php

```diff
@@ -24,19 +24,28 @@ public function parse(?string $text): ?string
             return null;
         }
 
-        $newText = preg_replace_callback(static::GJS_PRODUCTS_REGEX, static function (array $matches): string {
-            $gjsProductsDiv = str_replace(["\r", "\n", "\r\n"], '', $matches[0]);
-            preg_match('/data-products="(.+?)"/', $gjsProductsDiv, $productMatches);
-            $productArray = explode(',', $productMatches[1]);
-            $trimmedProductArray = array_map(static fn ($product) => trim($product), $productArray);
+        $crawler = new Crawler($text);
```

Comment:
This is awesome! :green_heart: 

Would it be too daring of me if I suggested there should be tests for `GrapesJsParser`? :smile:

Reply:
Tests rules! I fixed one error in implementation thanks to the test 💚

Reply:
awesome! :rocket: